manifests/latest/vsphere-7.0u1/vanilla/deploy/validatingwebhook.yaml

```diff
@@ -0,0 +1,35 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: sc-validation-webhook-svc
```

Comment:
The 'sc' prefix can be removed from the name, as the same service can be used for all admission validations, not only storage classes.

Reply:
+1

Reply:
fixed.